lightwood/helpers/general.py

```diff
@@ -120,21 +122,25 @@ def evaluate_array_accuracy(
     Scores are computed for each timestep (as determined by `timeseries_settings.nr_predictions`),
     and the final accuracy is the reciprocal of the average score through all timesteps.
     """
-
     ts_analysis = kwargs.get('ts_analysis', {})
     naive_errors = ts_analysis.get('ts_naive_mae', {})
```

Comment:
this one is tricky; is there a reason naive_errors is always mae?

Reply:
Yep, has to do with the definition of the MASE metric where these errors are used.